noir-projects/noir-contracts/contracts/app/orderbook_contract/src/test/utils.nr

```diff
@@ -0,0 +1,44 @@
+use crate::Orderbook;
+use aztec::{
+    protocol_types::address::AztecAddress, test::helpers::test_environment::TestEnvironment,
+};
+use token::Token;
+
+// Sets up test env with orderbook with two tokens and a minter account.
+// TODO(#16560): Make it possible to return a contract instance directly from setup func
```

Comment:
I feel like it might be worth tackling [this issue](https://github.com/AztecProtocol/aztec-packages/issues/16560) I just create soon to have a perfect reference. WDYT?